ci/infrastructure/scripts/infrastructure.sh

```
@@ -1101,14 +1105,12 @@ createBuildReport() {
     if [ ! -z "$VS_CONTAINER_EXT_PORT_SSH" ]; then
       echo "# SSH access (if enabled on the container) - available only on the Web Development LAN" | tee -a $VS_MAIL_NOTIFY_BUILD_MESSAGE
       echo "#   - ssh -o UserKnownHostsFile=/dev/null -p $VS_CONTAINER_EXT_PORT_SSH hippo@$VS_HOST_IP_ADDRESS ($VS_CONTAINER_SSH_PASS_HIPPO)" | tee -a $VS_MAIL_NOTIFY_BUILD_MESSAGE
-      echo "# " | tee -a $VS_MAIL_NOTIFY_BUILD_MESSAGE
     fi
     if [ -e "$VS_MAIL_NOTIFY_BUILD_MESSAGE_EXTRA" ]; then
-      cat $VS_MAIL_NOTIFY_BUILD_MESSAGE_EXTRA | tee -a $VS_MAIL_NOTIFY_BUILD_MESSAGE
+      cat $VS_MAIL_NOTIFY_BUILD_MESSAGE_EXTRA | grep -Ev "^$" | tee -a $VS_MAIL_NOTIFY_BUILD_MESSAGE
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)